### DISCUSSION TITLE
論亞洲的教育制度

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#1000] [General]
亞洲現在的教育制度說要多元學習，但是在本質上還是一樣維持著成績至上主義的想法，還是會有老師排斥成績不好的學生，邊愛成績比較好的學生的狀況發生，然後就是會考和學測，這種一考定終身的狀況也還是暗示著成績至上主義的情況，在美劇<菜鳥新移民中>主角一家人就是移民到美國的華裔人，很明顯地裡面的媽媽就是非常在乎成績，雖然只是一齣劇，但是也真實反饋著亞洲社會的現狀，真的很不喜歡這樣的教育制度，但是我也無能為力拉.....

**Comment by ru0372572** [comment]
我覺得觀念要改變其實是很難的，去跟上一輩的人聊天就知道了。不看成績是一回事，花時間去做的那些其他事情，姑且也希望能看到一定成果才能有辦法說服別人。

**Comment by delia07152** [comment]
成績至上的觀念已經根深蒂固了，要改變很難，雖然現在一直不斷有新的升學制度推出，但本質上還是在看成績。那你說會不公平嗎，我覺得倒也不會，至少我覺得目前最公平的的確是以成績為標準，至於其他新的制度，就等待它被提出吧，至少在這之前我們都還是得為了名次成績這些東西限制住。

**Comment by Tuoresa** [comment]
社會在快速評斷一個人的時候總是無法避免利用數字去衡量高低，無論是業績數據或是銷量成績等。這樣的現象也因此彰顯在學生身上，我們常會被成績所綑綁，造成彼此間的落差，這是無論什麼國家或身分都無法避免的比較。然而面對以成績至上觀念生活了超過半輩子的長輩們，以往能確保未來與生活保障最可靠的標準也無非是成績，需要他們在幾年內改變觀念真的非常困難。我想或許要能完全改變亞洲社會對於成績的想法，可能是要先消除亞洲教育「標準答案」的極致規範。相較於美國的大學入學考主要以測驗邏輯推理與概念為主，亞洲普遍的教育模式把「考試」這件事的好與壞都限制在分數高低之上，也就造成了成績之上的窒息風氣了。

**Comment by Sylvie0601** [comment]
我認為台灣的教育制度也是想要改革變成自由思考、開放性的學習這種教育模式，但根深蒂固的東方教育文化實在是沒辦法改變大家對於升學考試、成績的看中。

**Comment by Kotori0501** [comment]
我覺的以亞洲現在的教育制度是真的很難在改變了，確實有些老師會比較關注成績好的，但我覺得這部分已經改觀很多了，像我有一個老師比起成績更注重學生的個性與態度，所以平常遇到認識的老師真的要好好打招呼，說不定有一天他就會認識到你了

**Comment by bonny123865** [comment]
我也覺得注重品行跟態度的老師可遇不可求
之前有遇到幾個這樣的好老師就很幸運，不會因為成績不好就對同學酸言酸語的!

**Comment by Valentinaa0214** [comment]
亞洲的教育制度注重學習成果和考試成績，強調學生的學術能力，教育也是被視為進入社會的重要途徑，因此教育制度也會非常重視學生的就業能力和就業市場的需求。
但我認為它過度強調學術成績，忽略了學生的其他能力和發展。這種教育制度也造成度競爭和壓力過大，有時會導致學生的心理健康問題。雖然亞洲的教育制度在許多方面都取得了顯著的成就，但也存在一些問題和挑戰，需要不斷改進和完善。

**Comment by dabbie109303501** [comment]
過度只看成績固然不好 但考試制度已經是相對公平了
資源比不上 至少還有可能透過努力提高成績
當然有專業技能(技術)更重要
但是當一個人沒有技術 又沒有學歷能說服別人自己的才智 
憑什麼讓公司願意相信你能為他們創造價值呢

**Comment by WEI-CHEN-CHUN** [comment]
縱使如此至少我遇到的老師，都不太會排斥成績不好的學生

**Comment by AnAnistired** [comment]
我覺得我們都身處在這個亞洲教育的框架中，想要跳出去其實幾乎不可能，台灣教改改了又改，仍然不能擺脫成績至上主義的觀念，就算制度變了，人的想法沒變也是白搭

**Comment by Guagua1234** [comment]
雖然很多人認為亞洲的教育框架-成績為上已經根深蒂固，
而我認為成績不一定能定義一個人，
但成績也是某些人認真花費時間努力出來的，
每個人的想法、特長、想擁有的東西不一樣，
希望在未來不管是成績還是其他方面的努力都會被社會所認同。

**Comment by wenchi9023** [comment]
看成績來決定成敗我覺得適用於義務教育上，畢竟這真的是最公平的競爭方式
我不是天生就有聰明的頭腦，但我認為努力真的可以改變一個人的成就，就算沒辦法馬上體現，只要持續下去總會回饋到自己身上~
希望自己和大家也能早日脫離社會框架，找到自己的長處!

**Comment by Pietra1226** [comment]
雖然我不是教育家，也不太清楚其他亞洲的教育方式，但如果單以台灣來看，我覺得我們要跳脫成績框架，或是透過興趣學習本身就有難度。
因為台灣地狹人稠，產業本來就不夠多元，導致真正想一展長才的人無處發揮他的才能，而產業職業雖然沒有高低之分，但還是有高薪低薪之分，也真的比較少人會放棄追逐高薪。
在產業發展不夠廣跟薪水的誘因下，我認為家長在乎成績也不是沒有道理，當然如果移民了，還有這種想法就真的稍微過時了。
但這也不是說台灣完全沒辦法改變，只是需要時間，而且感覺需要很長一段時間。
等到我們捫心自問，我現在學的即使未來出路很爛，薪水很低，我還是要堅持。當大家都有這種想法的時候，自然而然台灣就能轉型成為一個不再追求成績的地方。

**Comment by Jasper525** [comment]
感覺台灣教育制度有一直想跟美國學習，只是改變需要階段性，一次改一點點。像現在高中的制度就跟我當時的有不小差距。

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
台灣現在的教學制度跟以往已經有很大的變化了 有慢慢往西方體制靠近 也不用一味的批評台灣體制

**Comment by Doreen331** [comment]
從小時候就一直被灌輸成績才是最重要的這個概念，所以一直拼命唸書沒發展什麼其他的興趣或才藝，但其實我現在長大之後回頭看，那些小時候被家長們認為成績不好的朋友們，也都在自己的道路上發展的越來越好，雖然不是很會唸書，但他們都找到自己的興趣過得很開心，相較我們這些在家長們眼中「步上正軌」的死大學生們，還在擔心畢業之後要幹嘛，我只能說每個選擇都有好跟壞，但最重要的是自己夠不夠努力

**Comment by Jzd314** [comment]
成績至上的概念還是在長輩中存在，每次家庭聚會都在比較誰家的小孩考的怎麼樣，但藉由成績進行分發學校感覺還是最公平的

**Comment by tower20021202** [comment]
成績至上的概念目前還是存在在大家的腦中，但很多時候當面試官要面試的人有成千上萬人的時候，成績往往會是你通往下一步的基本門檻，因此我認為雖然成績不是最重要的，但至少我們要能挺過這個基本門檻

**Comment by JessyCheng2003** [comment]
我也覺得在台灣有時候成績的影響蠻大的，之前有聽說過人資解釋為什麼連應徵工作都會看成績，尤其是理工方面，是因為從成績中可以看出學習態度，還有一點就是，公司想找的人是「乖乖聽話的員工」，守規矩且勤奮。

**Comment by zephyrxin01** [comment]
我覺得從小到大的升學制度、衡量你是不是好學生的標準都很難擺脫成績。這樣體制上的僵化也很難讓家長放心讓學生自由發展，如此一來老師也就很難改變教育的方法。

**Comment by alicia0725** [comment]
我認為目前為止台灣的教育制度暫時還不太可能完全放棄對成績的重視，但在近幾年來的升學制度已經加入了對多元表現要求的評分，雖然我沒有接觸到所謂的108課綱階段，但我覺得目前提供給學生參與的活動跟比賽等等在大學以前的階段非常不足，其實蠻難能夠在高中階段了解自己想學什麼

**Comment by alstinlin** [comment]
我覺得師長看中成績可能是因為教育方式比較保守，而且以成績當作進入學校或職場的門檻相對來說比較普遍，但如果能夠在其他領域發光發熱，讓他人看到，還是能夠為自己爭取到機會，所以無論是以學業或其他領域為目標，確立好自己努力的方向也很重要

**Comment by xiang305** [comment]
我認為透過108課綱，台灣的教育制度的重心已經不全然注重於成績上了，而是要去看學生在課堂之外的表現，不過目前的制度仍未完善。就我所知，我的高中同學透過個人申請進入大學之後，他們的教授表示他們不太會以多元表現的資料來評分，主因是時間花費太多。再者，多元表現資料要求高中生去參加科系相關的活動，變相壓縮了高中生選擇科系的時間，而且我也不明白為什麼已經有多元表現資料了還要再繳交一份自傳。總而言之，儘管目前的108課綱仍存在許多缺點，但政府的嘗試是值得鼓勵的，期待未來的教育制度會越來越好

**Comment by MaxChi870** [comment]
亞洲的教育體系幾乎都是成績至上主義，但近年來部分國家已在慢慢地改變這項觀念，將目光放到其他不同多元的表現，雖然這無法完全改變所有人的想法，但我仍覺得是個不錯的變化。

**Comment by ThanatosJun** [comment]
在每個人還未找到未來的定位時，我認為以成績評比，學測和考試等是符合多數利益的。在高中學生時期能找到自己想做甚麼的人當然是最好的，但是大多數來看是還沒找到！如果在高中時期就以特殊專長來為每一位學生進行分發，那麼哪項專長比較優？哪項專長又比較差呢？所以學科成績評比在高中階段我認為是最恰當的模式！個人看法。
